cg_hermes/config/balsamic.py

```diff
@@ -325,8 +297,6 @@
 }
 
 TUMOR_NORMAL_WGS_TAGS = {
-    frozenset({"normal-bam", "bam"}): {"is_mandatory": True},
-    frozenset({"normal-bam-index", "bam"}): {"is_mandatory": True},
     frozenset({"normal-cram", "cram"}): {"is_mandatory": True},
     frozenset({"cram", "normal-cram-index"}): {"is_mandatory": True},
     frozenset({"tumor-bam", "bam"}): {"is_mandatory": True},
```

Comment:
```suggestion

```
Also please remove `tumor-bam-index` in the next line

Reply:
done